src/SpotifyApi.NetCore/SpotifyWebApi.cs

```diff
@@ -172,8 +172,12 @@ private async Task<HttpResponseMessage> PostOrPut(string verb, Uri uri, object d
             _http.DefaultRequestHeaders.Authorization =
                 new AuthenticationHeaderValue("Bearer", accessToken ?? (await GetAccessToken()));
 
-            var content = data == null ? new StringContent("null") : new StringContent(JsonConvert.SerializeObject(data));
-            content.Headers.ContentType = new MediaTypeHeaderValue("application/json");
+            StringContent content = null;
+            if (data != null)
+            {
+                content = new StringContent(JsonConvert.SerializeObject(data));
+                content.Headers.ContentType = new MediaTypeHeaderValue("application/json");
+            }
```

Comment:
As discussed offline, this change is too big for this PR (it impacts almost every API class in the library). Create a new PR for this and write tests to assert that none of the other calls will break.